Python/sysmodule.c

```diff
@@ -2427,6 +2427,7 @@ static PyStructSequence_Field flags_fields[] = {
     {"inspect",                 "-i"},
     {"interactive",             "-i"},
     {"optimize",                "-O or -OO"},
+    {"use_peg",                 "-p old or -p new"},
```

Comment:
I would prefer to rename this field `_use_peg_parser`: repeat parser to make it easier to understand and make it private since it's going to be removed in Python 3.10.

`-p old or -p new` is outdated.

I would prefer to add the new field a the end.

Reply:
Done.